src/components/EmojiPicker/EmojiPickerButton.tsx

```diff
@@ -1,37 +1,40 @@
-import PropTypes from 'prop-types';
 import React, {useEffect, useRef} from 'react';
 import Icon from '@components/Icon';
 import * as Expensicons from '@components/Icon/Expensicons';
 import PressableWithoutFeedback from '@components/Pressable/PressableWithoutFeedback';
 import Tooltip from '@components/Tooltip/PopoverAnchorTooltip';
-import withLocalize, {withLocalizePropTypes} from '@components/withLocalize';
+import type { WithLocalizeProps } from '@components/withLocalize';
+import withLocalize from '@components/withLocalize';
 import withNavigationFocus from '@components/withNavigationFocus';
 import useStyleUtils from '@hooks/useStyleUtils';
 import useThemeStyles from '@hooks/useThemeStyles';
-import compose from '@libs/compose';
 import getButtonState from '@libs/getButtonState';
 import * as EmojiPickerAction from '@userActions/EmojiPickerAction';
+import type { Emoji } from '@assets/emojis/types';
 
-const propTypes = {
+type OnEmojiSelected = (emojiCode: string, emojiObject: Emoji) => void;
+
+type EmojiSelected = {
+    emojiCode: string;
+    emojiObject: Emoji;
+}
+type EmojiPickerButtonProps = WithLocalizeProps & {
```

Comment:
Not all params should be required here, throughout the PR please pay closer attention to prop-types (isRequired signals that param is required).